src/lib/containers/StackChanger.tsx

```diff
@@ -6,7 +6,10 @@ import { getStyleguideStack } from '../utils/SynapseClient'
 import { STYLEGUIDE_STACK_LOCAL_STORAGE_KEY } from '../utils/SynapseConstants'
 import { RadioGroup } from './widgets/RadioGroup'
 
-type Stack = 'production' | 'staging' | 'development'
+type Stack = 'production' | 'staging' | 'development' | 'mock'
```

Comment:
I didn't want to conflate production data with mocked data, so I added a new "mock" stack.